### PR TITLE
Fixed mkdocs theme settings.

### PR DESCRIPTION
**Pull request by jvanderaa**
# Closes: #4258 
# What's Changed

Adds to the theme copy code button based on https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button

Screenshot from 2023-08-14 14-48-22

# TODO

- [x] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [x] Attached Screenshots, Payload Example
- [x] Unit, Integration Tests
- [x] Documentation Updates (when adding/changing features)
- [x] Example Plugin Updates (when adding/changing features)
- [x] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ theme:
     - "search.suggest"
     - "search.highlight"
     - "search.share"
+    - "content.code.copy"
```
Comment (glennmatthews) [inline mkdocs.yml:23]: Feels like maybe this list should be alpha-sorted for maintainability?

File: changes/4258.fixed [hunk 2]
```diff
@@ -0,0 +1 @@
+Adds content.code.copy to the features list https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button
```
Comment (HanlinMiao) [inline changes/4258.fixed:1]: Please rename this file to 4258.added.
```suggestion
Added `content.code.copy` (https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button) to the `features` list in `mkdocs.yml`
```

**Comment by gsnider2195** [comment]
For context, the default behavior for this was changed to disabled in [Material for MkDocs v9.0.0](https://squidfunk.github.io/mkdocs-material/changelog/#9.0.0) which Nautobot upgraded to in v1.6.0.

File: changes/4258.fixed [hunk 3]
```diff
@@ -0,0 +1 @@
+Added `content.code.copy` (https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button) to the `features` list in `mkdocs.yml`
```
Comment (gsnider2195) [inline changes/4258.fixed:1]:
```suggestion
Re-enabled copy-to-clipboard button in mkdocs theme.
```

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**